dwave/preprocessing/composites/spin_reversal_transform.py

```diff
@@ -137,63 +137,100 @@ def _reorder_variables(sampleset: dimod.SampleSet,
                 will result in an independent run of the child sampler.
+                If srts is set then num_spin_reversal_transforms
+                is inferred by the shape, otherwise the default is 1.
 
         Returns:
             A sample set. Note that for a sampler that returns ``num_reads`` samples,
             the sample set will contain ``num_reads*num_spin_reversal_transforms`` samples.
 
+        Raises:
+            ValueError: If srts is inconsistent with num_spin_reversal_transforms or the bqm
+
         Examples:
-            This example runs 100 spin reversals applied to one variable of a QUBO problem.
+            This example runs 10 spin reversals applied to an unfrustrated chain of length 6
+
+            Using the returned lowest energy state (the ground state) returned, we can
+            define a special SRT that transforms all programmed couplers to be ferromagnetic 
+            (ground state to all 1).
 
             >>> from dimod import ExactSolver
+            >>> import numpy as np
             >>> from dwave.preprocessing.composites import SpinReversalTransformComposite
             >>> base_sampler = ExactSolver()
             >>> composed_sampler = SpinReversalTransformComposite(base_sampler)
             ...
-            >>> Q = {('a', 'a'): -1, ('b', 'b'): -1, ('a', 'b'): 2}
-            >>> response = composed_sampler.sample_qubo(Q,
-            ...               num_spin_reversal_transforms=100)
-            >>> len(response)
-            400
+            >>> num_var = 6
+            >>> num_spin_reversal_transforms = 10
+            >>> J = {(i, i+1): np.random.random() for i in range(num_var-1)}
+            >>> h = {i: 0 for i in range(num_var)}
+            >>> response = composed_sampler.sample_ising(h, J,
+            ...               num_spin_reversal_transforms=num_spin_reversal_transforms)
+            >>> len(response) == 2**num_var * num_spin_reversal_transforms
+            True
+            >>> SRT = np.array([response.first.sample[i]!=1 for i in range(num_var)])
+            >>> response = composed_sampler.sample_ising(h, J,
+            ...               srts=SRT[np.newaxis,:], num_reads=1)
+            >>> sum(response.record.num_occurrences) == 2**num_var
+            True
         """
         sampler = self._child
 
         # No SRTs, so just pass the problem through
-        if not num_spin_reversal_transforms or not bqm.num_variables:
+        if num_spin_reversal_transforms==0 or not bqm.num_variables:
             sampleset = sampler.sample(bqm, **kwargs)
             # yield twice because we're using the @nonblocking_sample_method
             yield sampleset  # this one signals done()-ness
             yield sampleset  # this is the one actually used by the user
             return
 
+
+        # Get the srts matrix
+        if srts is None:
+            # We maintain the Leap behavior that num_spin_reversal_transforms == 1
```

Comment:
I think this comment should be updated to reflect the fact Leap doesn't support/do SRTs anymore.